CedarJava/src/test/java/com/cedarpolicy/EntityTypeNameTests.java

```diff
@@ -36,7 +36,8 @@ public class EntityTypeNameTests {
         "else",
         "in",
         "like",
-        "has"
+        "has",
+        "is"
```

Comment:
luckily enough i ran into a test failure from this lol

Reply:
While it is nice that it will "eventually" catch the issue, but I really don't like non-determinism in CI. Created issue here: https://github.com/cedar-policy/cedar-java/issues/73